src/services/pncClient.ts

```diff
@@ -26,7 +26,7 @@ class PncClient {
     httpClient.interceptors.request.use((config) => {
       // perform actions before request is sent
 
-      if (keycloakService.isAuthenticated()) {
+      if (keycloakService.isKeycloakAvailable && keycloakService.isAuthenticated()) {
```

Comment:
if we care just about true or false, can't we move isKeycloakAvailable check into service? Same for getUser and hasRealmRole in appLayout? Seems like we would need to do this check in all classes when using those methods while they can return false from service when we don't need to differentiate the failure point.